lib/fhir_client/sections/crud.rb

```diff
@@ -147,12 +147,12 @@ def base_create(resource, options, format)
         options[:format] = format
         reply = post resource_url(options), resource, fhir_headers(options)
         if [200, 201].include? reply.code
-          type = reply.response[:headers][:content_type]
+          type = reply.response[:headers].select{|x| x.downcase=='content-type'}['content-type']
```

Comment:
`select` will also find all keys that match, `detect` is slightly better in this case since it will stop on the first one.